cmd/genji/shell/shell.go

```diff
@@ -370,6 +351,21 @@ func (sh *Shell) getAllTables() ([]string, error) {
 }
 
 func (sh *Shell) completer(in prompt.Document) []prompt.Suggest {
+	if strings.HasPrefix(in.Text, ".") {
+		expected := getAllCommands()
+		suggestions := make([]prompt.Suggest, len(expected))
+		for i, e := range expected {
+			suggestions[i].Text = e
+		}
```

Comment:
I think these 5 lines are the same regardless of the query, they should be done once and used here directly instead of being recreated every time a user runs a query.